files/build_templates/docker_image_ctl.j2

```diff
@@ -221,6 +221,10 @@ function postStartAction()
         # then we catch python exception of file not valid
         # that comes to syslog which is unwanted so wait till database
         # config is ready and then ping
+	# sonic-db-cli try to initialize the global database. If in multiasic platform, inital global
+        # database will try to access to all other instance database-config.json. If other instance
+        # database-config.json files are not ready yet, it will generate the sonic-db-cli core files.
+	waitForAllInstanceDatabaseConfigJsonFilesReady
```

Comment:
The crash is not cause by a behavior change, python version sonic-db-cli also crash in this case, access name space DB need name space config file ready.